api/swagger.yaml

```diff
@@ -1436,6 +1444,32 @@ paths:
         default:
           $ref: '#/responses/UnknownError'
 
+  /cisdockerbenchmarkresults/{id}:
+    get:
+      summary: 'Get CIS Docker Benchmark results by Application Resource'
+      parameters:
+        - $ref: '#/parameters/id'
+        - $ref: '#/parameters/cisDockerBenchmarkResultsSortKey'
+        - $ref: '#/parameters/sortDir'
+      responses:
+        '200':
+          description: 'Success'
+          schema:
+            type: 'object'
+            required:
+              - total
+            properties:
+              total:
+                type: 'integer'
+                description: 'Total CIS docker benchmark results by Application Resource'
+              items:
+                type: 'array'
+                description: 'List of CIS docker benchmark results by Application Resource'
```

Comment:
List of CIS docker benchmark findings for an Application Resource

Reply:
ack